tests/console/check_system_info.pm

```diff
@@ -63,4 +63,10 @@ sub run {
     }
 }
 
+sub post_fail_hook {
+    my $self = shift;
+    $self->SUPER::post_fail_hook;
+    upload_logs '/tmp/zypperlr.txt';
```

Comment:
I suggest to do your upload first then the base post_fail_hook, since sometimes the upload will fail in the base post_fail_hook.
so:
upload_logs '/tmp/zypperlr.txt';
$self->SUPER::post_fail_hook;